core/services/ardupilot_manager/main.py

```diff
@@ -290,6 +290,12 @@ async def available_boards() -> Any:
     return await autopilot.available_boards(True)
 
 
+@app.get("/safe")
```

Comment:
Doesn't make more sense to move to "armed" ? Safe is to abstracted, and armed is a common term.

Reply:
@patrickelectric I think `safe` can abstract more than just `arm`, in case we want to check for other things..?

Reply:
any idea of other things that the backend of ardupilot_manager may know ?
It would be nice to have a comment about that.

Reply:
if there's a flashing operation in progress, for example, I'll add comments and/or more checking

Reply:
comment added, I'll leave the actual blocking checks to after @JoaoMario109 's refactor is done